torch/csrc/jit/script/compiler.cpp

```diff
@@ -552,25 +552,68 @@ struct to_ir {
     emitLoopCommon({}, {cond}, stmt.body(), stmt, {});
   }
 
+  size_t calcNumStarredUnpack(const List<Expr>& lhs, const Assign& stmt) {
+    size_t num_normal_assign = 0;
+    size_t num_starred = 0;
+    for (const auto& assignee : lhs) {
+      if (assignee.kind() == TK_VAR) {
+        num_normal_assign++;
+      } else if (assignee.kind() == '*') {
+        num_starred++;
```

Comment:
Is `*something = expr` valid Python syntax? I don't think so, and it would be better if we tried to keep the script frontend close, unless we need it very badly

Reply:
Good point, this should throw if lhs.size() == 1